dto/src/main/scala/com/dwolla/cloudflare/domain/dto/ratelimits/RateLimitMatchDTO.scala

```diff
@@ -0,0 +1,26 @@
+package com.dwolla.cloudflare.domain.dto.ratelimits
+
+import com.dwolla.cloudflare.domain.dto.JsonWritable
+
+case class RateLimitMatchDTO (
+  request: RateLimitMatchRequestDTO,
+  response: Option[RateLimitMatchResponseDTO]
+) extends JsonWritable
+
+case class RateLimitMatchRequestDTO (
+  methods: Option[List[String]],
+  schemes: Option[List[String]],
+  url: String
+) extends JsonWritable
+
+case class RateLimitMatchResponseDTO (
+  status: Option[List[Int]],
+  origin_traffic: Option[Boolean],
```

Comment:
I didn't see this field listed in the Cloudflare API docs: https://api.cloudflare.com/#rate-limits-for-a-zone-rate-limit-details. Out of curiosity, is this something undocumented you encountered?

Reply:
Yeah, I've been using the fields documented here: https://api.cloudflare.com/#rate-limits-for-a-zone-properties

They seem to be more complete on what actually *could* be returned from the API.